django/test/testcases.py

```diff
@@ -303,6 +313,23 @@ def __call__(self, result=None):
         """
         self._setup_and_call(result)
 
+    def __getstate__(self):
+        """
+        Make SimpleTestCase pickable for parallel tests using subtests.
+        """
+        state = super().__dict__
```

Comment:
```suggestion
        state = super().__getstate__()
```

Reply:
On Python 3.10 you get an error
```
AttributeError: 'super' object has no attribute '__getstate__'
```

I guess only in 3.11, `object` class implements it by default: 
> Changed in version 3.11: Added the default implementation of the __getstate__() method in the [object](https://docs.python.org/3/library/functions.html#object) class.

Source: https://docs.python.org/3/library/pickle.html#object.__getstate__